src/main/java/org/jenkinsci/plugins/gitclient/ChangelogCommand.java

```diff
@@ -84,6 +84,18 @@
      */
     ChangelogCommand includes(ObjectId rev);
 
+    /**
+     * Sets the firstParent private variable to true
```

Comment:
I suggest to document the behaviour that is triggered by setting the variable value, not just that the variable is set. The former is more useful for the user of this API.